pkg/k8shandler/dashboards.go

```diff
@@ -0,0 +1,45 @@
+package k8shandler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+)
+
+const (
+	clusterLoggingDashboardFile = "dashboards/openshift-logging-dashboard.json"
+)
+
+// CreateOrUpdateDashboards reconciles metrics dashboards component for cluster logging
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateDashboards() (err error) {
+	if err = clusterRequest.createOrUpdateClusterLoggingDashboard(); err != nil {
+		return
+	}
+	return nil
+}
+
+// createOrUpdateClusterLoggingDashboard creates/updates the cluster logging dashboard ConfigMap
+func (clusterRequest *ClusterLoggingRequest) createOrUpdateClusterLoggingDashboard() error {
```

Comment:
why is this split from the public method?

Reply:
I literally copied this file from visualization.go and modified it's contents.